services/settings/pkg/config/config.go

```diff
@@ -45,4 +46,14 @@ type Metadata struct {
 	SystemUserID     string `yaml:"system_user_id" env:"OCIS_SYSTEM_USER_ID;SETTINGS_SYSTEM_USER_ID" desc:"ID of the oCIS STORAGE-SYSTEM system user. Admins need to set the ID for the STORAGE-SYSTEM system user in this config option which is then used to reference the user. Any reasonable long string is possible, preferably this would be an UUIDv4 format."`
 	SystemUserIDP    string `yaml:"system_user_idp" env:"OCIS_SYSTEM_USER_IDP;SETTINGS_SYSTEM_USER_IDP" desc:"IDP of the oCIS STORAGE-SYSTEM system user."`
 	SystemUserAPIKey string `yaml:"system_user_api_key" env:"OCIS_SYSTEM_USER_API_KEY" desc:"API key for the STORAGE-SYSTEM system user."`
+	Cache            *Cache `yaml:"cache"`
+}
+
+type Cache struct {
+	Store    string        `yaml:"store" env:"OCIS_CACHE_STORE;SETTINGS_CACHE_STORE" desc:"The type of the cache store. Supported values are: 'memory', 'ocmem', 'etcd', 'redis', 'redis-sentinel', 'nats-js', 'noop'. See the text description for details."`
+	Nodes    []string      `yaml:"addresses" env:"OCIS_CACHE_STORE_NODES;SETTINGS_CACHE_STORE_NODES" desc:"A comma separated list of nodes to access the configured store. This has no effect when 'memory' or 'ocmem' stores are configured. Note that the behaviour how nodes are used is dependent on the library of the configured store."`
+	Database string        `yaml:"database" env:"OCIS_CACHE_DATABASE" desc:"The database name the configured store should use."`
+	Table    string        `yaml:"table" env:"SETTINGS_CACHE_TABLE" desc:"The database table the store should use."`
```

Comment:
We need to be able to configure two tables here, as the service in setting up two caches (`dirs` and `files`)

Reply:
The `SETTINGS_CACHE_TABLE` has been implemented recently and after 2.0 so there is no legacy to take care of. You just could rename it to like `SETTINGS_CACHE_DIR_TABLE` and add a `SETTINGS_CACHE_FILE_TABLE`.

Reply:
Note that we need to adapt the settings README.md file to reflect caching.
You can take eg the frontend service readme as template.

Reply:
Uh, just seen, there is no readme.md but a `_index.md`...
You first need to copy the content of:
`docs/services/settings/_index.md` to `services/settings/README.md`.
I will help to fix the content as this definitely needs some care.

Reply:
> as this definitely needs some care.

Ok. Then let's please do the _index.md to README.md move (and content update) in a separate PR. Because those two changes should not block each other.